webapp/src/Controller/PublicController.php

```diff
@@ -116,6 +95,112 @@ public function scoreboardAction(Request $request): Response
         return $this->render('public/scoreboard.html.twig', $data, $response);
     }
 
+    /**
+     * @Route("/scoreboard-data.zip", name="public_scoreboard_data_zip")
+     */
+    public function scoreboardDataZipAction(RequestStack $requestStack, string $projectDir, Request $request): Response
+    {
+        $contest = $this->getContestFromRequest($request) ?? $this->dj->getCurrentContest(-1);
```

Comment:
no access check required or is this implicit?

Reply:
The only check is for non public scoreboards and that should be implicit, but I will verify

Reply:
Indeed, `getCurrentContest` (which is also used in `getContestFromRequest`) checks whether you have access.